app/assets/javascripts/pages/scenario-form-page.js.coffee

```diff
@@ -0,0 +1,13 @@
+class @ScenarioFormPage
+  constructor:() ->
+    @format
+    @enabledSelect2
```

Comment:
`@enabledSelect2` will only return the function definition, to actually call it the parentheses are required like in plain javascript.